content/news/XXXX-XX-XX-github-issues-migration.md

```diff
@@ -0,0 +1,41 @@
+title: Migrated to Github issues
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Yes everybody, we're officially moving away from Launchpad to Github.
+If you've never tried to report a Bug in Mixxx or wanted to contribute
+otherwise, its very likely you have no idea what this Post is about,
+but if you do its even better. No matter if you do or don't, keep reading.
+
+The mixxx team has traditionally been using canonicals (the makers of Ubuntu)
+launchpad platform for hosting their code, managing bug reports and releases.  
+Over the years, launchpad has become increasingly dated, lack of modern features,
+old UI, and bad user experience is the primary reason why many people (myself
+included), especially new potential contributors, stay away from launchpad.
+We suspect Launchpad to be one of the major deterents for potential mixxx contributors.
+We've started transitioning away from launchpad quite a while ago.
+<!-- The rest of this paragraph needs further info from older mixxx team members.
+As I was not around when this happened, most of what I'm writing here is pure speculation
+from my part. Please correct me if I'm wrong or if I should leave this out completely.-->
+
+Back in XXXX we made the first major step away from Launchpad by migrating the
+mixxx source code to github. Still, bug reports continued to be handled on launchpad.
+TODO
+
+TODO launchpad just used for the ppa, is that correct? no contributor interaction on launchpad
+anymore.
+
+TODO write about how we migrated all issues to mixxx
```

Comment:
At one point we make the bug tracking in Launchpad read-only and run the migration. The Lauchpad tags will be migrated 1 to 1. We drop the priorities, because in a volunteer project like Mixxx we have no control about the personal priorities of each contributor.  The remaining Launchpad bug states and priorities will be mapped to special tags as in the table below: 

| Launchpad  | GitHub |
| ------------- | ------------- |
| Confirmed | confirmed |
| Fix Committed | (close)  |
| Fix Released | (close) |
| Incomplete | incomplete |
| In Progress | |
| Invalid | invalid |
| New | |
| Triaged | confirmed |
| Won't Fix | wontfix |
| Critical | party stopper; bug |
| High | bug |
| Low | bug |
| Medium | bug |
| Undecided |  |
| Wishlist | feature |

Reply:
I am aware about that table. The question is whether we should mention the "transformations" done as part of the migration in this blogpost.

Reply:
> We drop the priorities, because in a volunteer project like Mixxx we have no control about the personal priorities of each contributor.

I'm not sure if I agree with dropping these, but I'll continue that discussion on zulip.

Reply:
Nevermind, I just saw that we do have labels for the `Critical` and `Wishlist` priorities. Its fine for me that we can drop the rest.

Reply:
So do you propose to add the table to this blogpost?

Reply:
Sure